packages/forms/src/components/Repeater/Repeater.tsx

```diff
@@ -73,7 +73,7 @@ const Fieldset = ({
     );
 
   const disableIncrement =
-    isLast || // is last item
+    isLast ?? // is last item
```

Comment:
I think this should be ||.

Reply:
Updated in [ignore boolean primitives](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11651/commits/ccb9ba3ec73672fea328bb97fecdb8b1c1db5b91)